strings/word_occurence2.py

```diff
@@ -8,8 +8,8 @@
 def word_occurence(sentence: str) -> dict:
     """
     >>> from collections import Counter
-    >>> all(word_occurence(s) == Counter(s.split()) for s in
-    ...     ("a b a b c a", "I love to love Python ❤️ 💕 ❤️", ""))
+    >>> all(word_occurence(s) == Counter(s.split(' ')) for s in
+    ...     ("a b a b c a", "I love to love Python ❤️ 💕 ❤️", "", " ; ;"))
```

Comment:
The code puts out the wrong output when the input is an empty string.

Reply:
Then I should change line 16 to `.split()`. Otherwise, it does count "empty" char as a string tho:
```
python3 -i word_occurence2.py 
Input: 
: 1
>>> word_occurence('')
{'': 1}
```
Thanks for the suggestions.